### ISSUE TITLE
Kernel crashes on INT21,6521, CX=0

### ISSUE THREAD

**Issue by boeckmann** [SvarDOS/edrdos#41]
According to RBIL [1], Novel DOS 7 update 15 crashes on the above condition. Looking at the current source https://github.com/SvarDOS/edrdos/blob/9751c114b84df883956fa289a0142dfe54b57854/drdos/misc.a86#L1319-L1333
this still seems to be the case.

[1] http://www.ctyme.com/intr/rb-3164.htm

**Comment by ecm-pushbx** [comment]
Looks like you just need a `jcxz` before f6521_10 that jumps to the ret.

**Comment by boeckmann** [comment]
Yes :-) https://github.com/SvarDOS/edrdos/commit/287efb457c36e7bd51fe245ed6b1bd43d732a48d
